# Name of all end effectors involved in the pose
string[] end_effector_link_names
# Poses of each end effector
geometry_msgs/PoseArray pose_array
